java/semantickernel-api/src/main/java/com/microsoft/semantickernel/templateengine/handlebars/HandlebarsPromptTemplate.java

```diff
@@ -299,4 +244,44 @@ public Mono<String> render(KernelFunctionArguments variables) {
             }
         }
     }
+
+    @SuppressWarnings("StringSplitter")
+    private static void addFunctionHelpers(Kernel kernel, Handlebars handlebars) {
+        kernel
+            .getPlugins()
+            .forEach(plugin -> {
+                plugin
+                    .iterator()
+                    .forEachRemaining(kernelFunction -> {
+                        String functionName = kernelFunction.getName();
+                        String pluginName = plugin.getName();
+                        handlebars.registerHelper(
+                            pluginName + "-" + functionName,
```

Comment:
This is an oft repeated pattern. We should have either a static method on plugin that gives back the canonical plugin plus function name, or a getCanonicalName method on KernelFunction itself (which would mean a KernelFunction would have to know what plugin it belongs to).